CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -96,12 +98,13 @@ public void TearDown()
         [Test]
         public void CanRemindLateDevelopersAtTenThirtyOnFriday()
         {                      
-            var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
             var clock = new ClockStub(
                 new DateTimeOffset(
                     new DateTime(2019, 03, 01, 10, 30, 0)
                 )
             );
+            
+            var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway, clock);
```

Comment:
This line is duplicated and could perhaps be extracted into a `SetUp` within a `TestFixture`.

Reply:
No, it cannot go to the SetUp method because the clock changes.